__tests__/index.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { main, InputValues } from '../src';
+import { main, INPUT } from '../src';
```

Comment:
What was the reason for this, stylistic? It seems to generate a load of change unrelated to the original issue so just curious why.

Reply:
I often declare properties that I have to use across a program with enums, this is because I try to avoid magical strings (I also make mistakes) when I have an enum I can always refer to it it doesn't matter what the property really is. 

The issue is that JS doesn't allow you to declare keys as kebab without having to quote and use the bracket notation 
`object['key-that-is-awkward-for-js']`. when I started changing to the kebab case, many of the ways I was retrieving keys will not work. a common example:
I need to get all the values provided by the user: 
before
`const {PROPERTY = DEFAULT_VALUE, ...} = getInput();` 
after, because the value is kebab, I will have a runtime error if I do:
`const {kebab-case-variable = DEFAULT_VALUE, ...} = getInput();` 
so I have to use a different notation which produced a bunch of type changes a long the way.